content/blog/culture-war-open-source.md

```diff
@@ -0,0 +1,188 @@
++++
+title = ""The Ethical Source Working Group Blog"
+heading = "The Culture War in Open Source is On"
++++
+
+The challenges to open source\'s \"noble neutrality.\"
+
+[Nathan Schneider](https://nathanschneider.info/)   
+April 15, 2020
+
+Perhaps you remember Eric S. Raymond from his memorable essay "[The
+Cathedral and the
+Bazaar](http://www.catb.org/esr/writings/cathedral-bazaar/)," or his[
```

Comment:
It looks like you have something weird going on with your links. I'm seeing this:

```[
](http://catb.org/~esr/faqs/smart-questions.html#not_losing)[advice](http://catb.org/~esr/faqs/smart-questions.html#not_losing)```
```
(notice how there is an empty anchor, a URL, then an anchor, and the same URL?

Should be like this:

```[advice](http://catb.org/~esr/faqs/smart-questions.html#not_losing)```

And it's showing as raw links. I think for links it should be just `[anchor text](http://…)`

Reply:
Eeek sorry! I didn't notice that got screwed up in the pandoc conversion. Will fix.